staging/src/k8s.io/apiserver/pkg/storage/errors.go

```diff
@@ -95,6 +105,10 @@ func IsNotFound(err error) bool {
 	return isErrCode(err, ErrCodeKeyNotFound)
 }
 
+func IsTooLargeResourceVersion(err error) bool {
```

Comment:
can we document this func like line 112 below it?

Reply:
Good catch. Fixed.